geometric_kernels/sampling/feature_maps.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
I'd suggest to move `feature_maps.py` out of the `sampling` folder because feature maps are not exclusively used for sampling

Reply:
/kernels/ is a better location, or the root